site/cypress/e2e/Cypress-Feature/discussion_post.spec.js

```diff
@@ -0,0 +1,27 @@
+describe('test for discussion post panel', () => {
+    it('Enable discussion form and post', () => {
+        cy.login();
+        cy.visit(['sample', 'config']);
+        cy.get('[data-testid="forum-enabled"]').should('be.checked');
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'update']);
+        cy.get('[data-testid="yes-discussion"]').click();
+        cy.get('[data-testid="yes-discussion"]').should('be.checked');
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'grade?who_id=hG1b13ljpDjKu32&sort=id&direction=ASC']);
+        cy.get('[data-testid="discussion-browser-btn"]').type('{D}');
+        cy.get('[data-testid="posts-list"]').should('contain', 'Discussion Posts');
+        cy.get('[data-testid="posts-list"]').should('contain', 'No thread id specified.');
+        cy.get('[data-testid="posts-list"]').should('not.contain', ' Go to thread');
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'update']);
+        cy.get('[data-testid="discussion-thread-id"]').type('1,2,3');
+        cy.get('[data-testid="discussion-thread-id"]').type('{enter}');
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'grade?who_id=hG1b13ljpDjKu32&sort=id&direction=ASC']);
+        cy.get('[data-testid="posts-list"]').should('contain', 'No posts for thread id: 1');
+        cy.get('[data-testid="create-post-head"]').first().should('contain', '(2) Homework 1 print clarification');
+        cy.get('[data-testid="create-post-head"]').eq(1).should('contain', '(3) Homework 1 not running');
+        cy.get('[data-testid="posts-list"]').should('contain', ' Go to thread');
+        cy.get('[data-testid="go-to-thread"]').its('length').should('eq', 3);
+        cy.get('[data-testid="go-to-thread"]').eq(1).click({force: true});
```

Comment:
is this force click necessary?

Reply:
Oh I think I forget to remove it , when I click on button go to thread it opened that page in some other tab but later I removed target="_blank" 
Will fix it

Reply:
Tested locally again , yeah we need to defined , without `force:true` it is failing

Reply:
I haven't tested this locally, but if the element isn't visible, we shouldn't be able to click on it (because you can't manually I think. with a few exceptions)

Reply:
it looks good to me in this way 

`cy.get('[data-testid="go-to-thread"]').eq(1).should('have.attr', 'href').and('contain', 'sample/forum/threads/2');`

Reply:
We are trying to remove the force clicks.